### PR TITLE
Fix footer position and Input radius

### PR DESCRIPTION
**Pull request by evulins**
Fix styles in Footer and Input components.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { FC } from "react";
 
 import Navbar from "./components/atoms/Navbar";
 import Footer from "./components/organisms/Footer";
-import ProblemIndex from "./components/pages/ProblemIndex";
+import ProblemIndex from "./components/ProblemIndex";
```
Comment (franiakat) [inline src/App.tsx]: We want to keep `pages` within `components`. ;)

File: src/components/molecules/SearchAutocomplete/index.tsx [hunk 2]
```diff
@@ -69,6 +70,7 @@ const SearchAutocomplete: FC<SearchAutocompleteProps> = ({ suggestions }) => {
     >
       <S.Form
         // autoComplete="off"
```
Comment (franiakat) [inline src/components/molecules/SearchAutocomplete/index.tsx]: I think this isn't needed in the end, we can remove it here as it's tiny. :)

**Comment by bazelajoanna** [comment]
🥇 great!

File: src/components/molecules/SearchAutocomplete/interface.ts [hunk 3]
```diff
@@ -11,3 +11,7 @@ export interface SuggestionsProps {
 export interface SuggestionProps {
   isSelected?: boolean;
 }
+
+export interface InputProps {
+  showSuggestions: boolean;
```
Comment (franiakat) [inline src/components/molecules/SearchAutocomplete/interface.ts]: I am wondering whether to indicate those are Style related props, like we did in Button case.
```suggestion
export interface InputStyleProps {
  showSuggestions: boolean;
```

**Review by franiakat — COMMENTED** [review]
Looks marvellous 🌟 Just those tiny question marks to cover. : >

File: src/App.tsx (hunk 1 above)
Comment (evulins) [reply src/App.tsx]: For sure we want this, i don't know what happened here

File: src/components/molecules/SearchAutocomplete/index.tsx [hunk 4]
```diff
@@ -69,8 +69,6 @@ const SearchAutocomplete: FC<SearchAutocompleteProps> = ({ suggestions }) => {
       aria-haspopup="listbox"
     >
       <S.Form
-        // autoComplete="off"
-        showSuggestions={showSuggestions}
```
Comment (franiakat) [inline src/components/molecules/SearchAutocomplete/index.tsx]: This one stays I guess. 🙃

Comment (evulins) [reply src/components/molecules/SearchAutocomplete/index.tsx]: I don't think so :D

Comment (evulins) [reply src/components/molecules/SearchAutocomplete/index.tsx]: Hit refresh

Comment (franiakat) [reply src/components/molecules/SearchAutocomplete/index.tsx]: ahaha this is a Form component :D NVM

Comment (evulins) [reply src/components/molecules/SearchAutocomplete/index.tsx]: I forgot to delete it earlier

**Review by franiakat — APPROVED** [review]
sqash

**Pull request merged**